config/server.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Server port="8005" shutdown="SHUTDOWN">
+    <Listener className="org.apache.catalina.startup.VersionLoggerListener" />
+    <Listener className="org.apache.catalina.core.AprLifecycleListener" SSLEngine="on" />
+    <Listener className="org.apache.catalina.core.JreMemoryLeakPreventionListener" />
+    <Listener className="org.apache.catalina.mbeans.GlobalResourcesLifecycleListener" />
+    <Listener className="org.apache.catalina.core.ThreadLocalLeakPreventionListener" />
+
+    <GlobalNamingResources>
+        <Resource
+            name="UserDatabase"
+            auth="Container"
+            type="org.apache.catalina.UserDatabase"
+            description="User database that can be updated and saved"
+            factory="org.apache.catalina.users.MemoryUserDatabaseFactory"
+            pathname="conf/tomcat-users.xml"
+        />
+    </GlobalNamingResources>
+
+    <Service name="Catalina">
+        <Connector port="8080" protocol="HTTP/1.1"
+            connectionTimeout="20000"
+            redirectPort="8443"
+            URIEncoding="UTF-8"
+            relaxedQueryChars='\ { } | [ ]'
+        />
+
+        <Connector port="8009" protocol="AJP/1.3" redirectPort="8443" />
+
+        <Engine name="Catalina" defaultHost="localhost">
+            <Realm className="org.apache.catalina.realm.LockOutRealm">
+                <Realm
+                    className="org.apache.catalina.realm.UserDatabaseRealm"
+                    resourceName="UserDatabase"
+                />
+            </Realm>
+
+            <Host
+                name="localhost"
+                appBase="webapps"
+                unpackWARs="true"
+                autoDeploy="false"
+                deployOnStartup="false"
+            >
+                <Context path="${context.path}" docBase="ROOT/" />
```

Comment:
This *might* (not tested yet) produce issues with windows host machines. Will test and report back.